File: TransCoder_tokenized_test_set_functions/python/MAXIMUM_DISTINCT_LOWERCASE_ALPHABETS_TWO_UPPERCASE.py
def maxLower ( str ) :
    n = len ( str )
    i = 0
    for i in range ( n ) :
        if str [ i ] >= 'A' and str [ i ] <= 'Z' :
            i += 1
            break
    maxCount = 0
    count = [ ]
    for j in range ( MAX_CHAR ) :
        count . append ( 0 )
    for j in range ( i, n ) :
        if str [ j ] >= 'A' and str [ j ] <= 'Z' :
            currCount = 0
            for k in range ( MAX_CHAR ) :
                if count [ k ] > 0 :
                    currCount += 1
            maxCount = max ( maxCount, currCount )
            for y in count :
                y = 0
        if str [ j ] >= 'a' and str [ j ] <= 'z' :
            count [ ord ( str [ j ] ) - ord ( 'a' ) ] += 1
    return maxCount

File: TransCoder_tokenized_test_set_functions/java/MAXIMUM_DISTINCT_LOWERCASE_ALPHABETS_TWO_UPPERCASE.java
static int maxLower ( String str ) {
  int n = str . length ( );
  int i = 0;
  for (;
  i < n;
  i ++ ) {
    if ( str . charAt ( i ) >= 'A' && str . charAt ( i ) <= 'Z' ) {
      i ++;
      break;
    }
  }
  int maxCount = 0;
  int count [ ] = new int [ MAX_CHAR ];
  for (;
  i < n;
  i ++ ) {
    if ( str . charAt ( i ) >= 'A' && str . charAt ( i ) <= 'Z' ) {
      int currCount = 0;
      for ( int j = 0;
      j < MAX_CHAR;
      j ++ ) {
        if ( count [ j ] > 0 ) {
          currCount ++;
        }
      }
      maxCount = Math . max ( maxCount, currCount );
      Arrays . fill ( count, 0 );
    }
    if ( str . charAt ( i ) >= 'a' && str . charAt ( i ) <= 'z' ) {
      count [ str . charAt ( i ) - 'a' ] ++;
    }
  }
  return maxCount;
}